content/en/examples/admin/dns/dns-horizontal-autoscaler.yaml

```diff
@@ -1,33 +1,94 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: kube-dns-autoscaler
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Why this annotation?

Reply:
Addons with label addonmanager.kubernetes.io/mode=Reconcile will be periodically reconciled. Direct manipulation to these addons through apiserver is discouraged because addon-manager will bring them back to the original state. In particular:

- Addon will be re-created if it is deleted.
- Addon will be reconfigured to the state given by the supplied fields in the template file periodically.
- Addon will be deleted when its manifest file is deleted from the $ADDON_PATH.

https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/README.md#addon-manager

Reply:
@sftim does this answer your question adequately or is this still a blocking concern?

Reply:
(it's not a blocking concern as such)

I wondered about it because it's apparently an official annotation but not mentioned in [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/). If we add this annotation to official documentation, that is tantamount to blessing it as official even though the add-on manager is in fact deprecated / unofficial (everything in https://github.com/kubernetes/kubernetes/blob/master/cluster/ is deprecated).